manager/sql/databases.go

```diff
@@ -178,6 +179,30 @@ var Migrations = map[string]Statements{
 			(subject.has_regex IS NOT TRUE AND subject.template = $1)
 			OR
 			(subject.has_regex IS TRUE AND $2 ~ subject.compiled)`,
+		QueryPoliciesForResource: `
```

Comment:
This is pretty much the same query as before - maybe this could be a helper function with `Sprintf` which replaces the `subject.has_regex` part with `%s.has_regex` and so on?

Reply:
Done. I put it in a separate commit because I felt not-completely sure that it is an upgrade.
It definitely removes a lot of repetition, but maybe it slightly obfuscates the SQL.